scripts/sanitise-db.sql

```diff
@@ -337,19 +340,50 @@ $$ LANGUAGE plpgsql;
 
 DO $$
 BEGIN
-    PERFORM anonymize_with_lorem_ipsum('funding', 'data_blob', 'funding_source');   --- LARGE TABLE
+    PERFORM anonymize_with_lorem_ipsum('funding', 'data_blob', 'funding_source');
+    PERFORM anonymize_with_lorem_ipsum('funding', 'data_blob', 'spend_type');
+
     PERFORM anonymize_with_lorem_ipsum('funding_comment', 'data_blob', 'comment');
+
+    PERFORM anonymize_with_lorem_ipsum('funding_question', 'data_blob', 'question');
+    PERFORM anonymize_with_lorem_ipsum('funding_question', 'data_blob', 'indicator');
+    PERFORM anonymize_with_lorem_ipsum('funding_question', 'data_blob', 'response');
+    PERFORM anonymize_with_lorem_ipsum('funding_question', 'data_blob', 'guidance_notes');
+
+    PERFORM anonymize_int_value_with_percentage_range('outcome_data', 'amount', -20, 20);
+    PERFORM anonymize_int_value_with_percentage_range('output_data', 'amount', -20, 20);
+    PERFORM anonymize_with_lorem_ipsum('output_data', 'data_blob', 'additional_information');
+
     PERFORM anonymize_with_lorem_ipsum('private_investment', 'data_blob', 'additional_comments');
+    PERFORM anonymize_int_value_with_percentage_range('private_investment', 'townsfund_funding', -20, 20);
+    PERFORM anonymize_int_value_with_percentage_range('private_investment', 'total_project_value', -20, 20);
+
+    PERFORM anonymize_with_lorem_ipsum('programme_funding_management', 'data_blob', 'payment_type');
+
     PERFORM anonymize_with_lorem_ipsum('programme_progress', 'data_blob', 'answer');
 
-    PERFORM anonymize_postcode_in_column('project_dim', 'postcodes');
+    PERFORM anonymize_with_lorem_ipsum('project_progress', 'data_blob', 'commentary');
+    PERFORM anonymize_with_lorem_ipsum('project_progress', 'data_blob', 'important_milestone');
+    PERFORM anonymize_with_lorem_ipsum('project_progress', 'data_blob', 'leading_factor_of_delay');
+    PERFORM anonymize_with_lorem_ipsum('project_progress', 'data_blob', 'delivery_status');
+    PERFORM anonymize_with_lorem_ipsum('project_progress', 'data_blob', 'delivery_stage');
+    PERFORM anonymize_with_lorem_ipsum('project_progress', 'data_blob', 'adjustment_request_status');
+    PERFORM anonymize_int_value_with_percentage_range('project_progress', 'risk_rag', -20, 20);
```

Comment:
Good spot by @samuelhwilliams on this - RAG ratings are between 1 and 5 I believe so best to use that as the range for a realistic representation. Might be worth double checking the code to see if it is 1 to 5 and not some other range or some text value, I can point you to the right places if you are unsure